instructionAPI/src/AMDGPU/gfx908/amdgpu_gfx908_decoder_impl.h

```diff
@@ -1,5 +1,3 @@
-#include <stdint.h>
```

Comment:
Did you update your generation scripts to include Jim's updates for including standard headers?

Reply:
Resolved.